include/ceed/ceed.h

```diff
@@ -624,6 +621,15 @@ CEED_EXTERN int CeedQFunctionFieldGetSize(CeedQFunctionField qf_field,
 CEED_EXTERN int CeedQFunctionFieldGetEvalMode(CeedQFunctionField qf_field,
     CeedEvalMode *eval_mode);
 
+/// Handle for object describing CeedQFunctionContext fields
+/// @ingroup CeedQFunction
+typedef struct {
+  const char *name;
+  const char *type;
```

Comment:
Better as an enum. Can have an array of names indexed by the enum values for pretty printing.

Reply:
Agreed, I was just being quick/lazy to get the idea running